src/data/bucket/symbol_bucket.js

```diff
@@ -598,7 +598,7 @@ class SymbolBucket implements Bucket {
         }
     }
 
-    addToLineVertexArray(anchor: Anchor, line: any) {
+    addToLineVertexArray(anchor: Anchor, line: any): {| lineLength: number, lineStartIndex: number |} {
```

Comment:
What style should we follow concerning inlined anonymous type? I sometimes find it nice to read a named type instead (e.g. `addToLineVertexArray(anchor: Anchor, line: any): LineEntry`) as a hint to what it is. This can also reduce very long function signature like [here](https://github.com/mapbox/mapbox-gl-js/blob/main/src/symbol/placement.js#L332).

We don't have to switch all of the ones we already have, but since we're going to add a lot of these now I wonder if we should adopt a particular convention.

Reply:
Good point — this return type was auto-generated, but I agree that extracting it into a named type makes it more readable (and perhaps it will be useful in the future once we need to specify this type in other files). Fixed.